docs/installation/bootstrap.rst

```diff
@@ -120,31 +141,17 @@ Bootstrap node.
     destination fields of IP packets to correspond to the MAC address(es))
     :ref:`IP-in-IP needs to be enabled<enable IP-in-IP>`.
 
-Provision storage for Prometheus services
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-After bootstrapping the cluster, the Prometheus and AlertManager services used
-to monitor the system will not be running (the respective :term:`Pods <Pod>`
-will remain in *Pending* state), because they require persistent storage to be
-available. You can either provision these storage volumes on the bootstrap
-node, or later on other nodes joining the cluster. Templates for the required
-volumes are available in :download:`examples/prometheus-sparse.yaml
-<../../examples/prometheus-sparse.yaml>`. Note, however, these templates use
-the `sparseLoopDevice` *Volume* type, which is not suitable for production
-installations. Refer to :ref:`volume-management` for more information on how to
-provision persistent storage.
-
-.. note::
-
-   When deploying using Vagrant, persistent volumes for Prometheus and
-   AlertManager are already provisioned.
-
 Validate the install
 ^^^^^^^^^^^^^^^^^^^^
-Check if all :term:`Pods <Pod>` on the Bootstrap node are in the
-``Running`` state.
+- Check if all :term:`Pods <Pod>` on the Bootstrap node are in the
```

Comment:
If we don't mention provisioning storage for Prometheus and such before this, then they *can't* be all in `Running` state.